### PR TITLE
AEROGEAR-10181 - Integrate dynamic DOC_LINKS from UPS SERVER

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-10181

## What
* Now the AdminUI reads the UPS_SERVER from the `config` file in the `public` folder
* Now the AdminUI calls the `/rest/ui/config` endpoint to have the list of URLs

### CODE REVIEW

File: src/application/ApplicationDetail/SenderAPI.tsx [hunk 1]
```diff
@@ -40,7 +41,20 @@ export class SenderAPI extends Component<Props, State> {
     };
   }
 
+  async componentDidMount() {
+    this.setState({ docLinks: await Config.getInstance().getDocsConfig() });
+  }
+
```
Comment (secondsun) [inline src/application/ApplicationDetail/SenderAPI.tsx:47]: This code and getLinks() below is duplicated a lot. Should the links be something that should be at the context level?

File: src/application/ApplicationDetail/SenderAPI.tsx [hunk 2]
```diff
@@ -126,13 +140,13 @@ export class SenderAPI extends Component<Props, State> {
             />
             <Text component={'small'}>
               Read more on the details of the{' '}
-              <a href={links.pushApplications.senderAPI.docs.java_client}>
+              <a href={getLink('sender-api-java')}>
```
Comment (secondsun) [inline src/application/ApplicationDetail/SenderAPI.tsx:143]: There's an old style link above at 

https://github.com/aerogear/unifiedpush-admin-ui/blob/743b6730b0572c37f7a771368e1bddb2f9f61667/src/application/ApplicationDetail/SenderAPI.tsx#L127

File: src/application/crud/CreateVariantPage.tsx [hunk 3]
```diff
@@ -44,14 +59,7 @@ export class CreateVariantPage extends Component<Props, State> {
...
             device. Learn more about variants in the{' '}
-            <a
-              href={
-                'https://docs.aerogear.org/aerogear/latest/getting-started-running.html#running'
-              }
-            >
-              {' '}
-              documentation{' '}
-            </a>
+            <a href={getLink('getting-started#running')}> documentation </a>
```
Comment (secondsun) [inline src/application/crud/CreateVariantPage.tsx:62]: I couldn't figure out where this mapped, in app it gives an empty link

File: src/application/wizard/SetupPage.tsx [hunk 4]
```diff
@@ -116,7 +132,7 @@ export class SetupPage extends Component<Props, State> {
               Next we are going to send a test notification. Make sure you {''}
               <Text
                 component={TextVariants.a}
-                href="https://aerogear.org/docs/unifiedpush/aerogear-push-android/guides/#_handling_notification"
+                href={getLink('build-and-deploy-android')}
```
Comment (secondsun) [inline src/application/wizard/SetupPage.tsx:135]: `build-and-deploy-android` seems to be used for all variants (android, ios, etc) should it be renamed?

File: src/application/crud/CreateVariantPage.tsx (hunk 3 above)
Comment (ziccardi) [reply src/application/crud/CreateVariantPage.tsx:57]: Probably you need to update your UPS server. It has been added here: https://github.com/aerogear/aerogear-unifiedpush-server/pull/1118/files#diff-8154e370aeedaebfe91fcc0487a5f659R3

**Review by secondsun — APPROVED** [review]

**Pull request merged**